src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c

```diff
@@ -421,15 +421,20 @@ static int xclbin_load_axlf(struct platform_device *pdev, const void *buf)
 	}
 	xocl_subdev_destroy_by_level(xdev, XOCL_SUBDEV_LEVEL_URP);
 
+	ret = xocl_axigate_freeze(xdev, XOCL_SUBDEV_LEVEL_PRP);
+	if (ret)
+		goto done;
```

Comment:
will this break the legacy platform support? I don't think we have axigate subdriver there.

Reply:
yes, do just ignore the return value. The axigate freeze and free will only return 0 or -ENODEV;.